### PR TITLE
Add meteoroid overview, architecture, feature documentation

### PR DESCRIPTION
**Pull request by hide-hayashi**
Hi, I have written meteoroid overview and architecture documentation.
Please give me a review.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,22 +3,24 @@
 ![PyPI](https://img.shields.io/pypi/v/meteoroid-cli)
 ![GitHub](https://img.shields.io/github/license/OkinawaOpenLaboratory/fiware-meteoroid?color=blue)
 
-## Overview
+## Welcome to Meteoroid Documentation
```
Comment (YujiAzama) [inline docs/index.md]: I think this header should be at the top of the page and a top-level header(h1).

**Review by YujiAzama — COMMENTED** [review]
Do you match the image file names to the hierarchy?
If so, please change the name of the image `home-component.png` to `architecture-component.png`.
Architecture is not under Home.

And please correct the spelling of image name `architecure` and `comportment`.

File: docs/architecture.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Architecture
+
+Meteoroid provides users with a function to centrally manage FIWARE Orion's Subscription function and Apache OpenWhisk's FaaS function in order to realize an event-driven architecture in FIWARE.
+
+Meteoroid consists of OpenWhisk and Meteoroid core. OpenWhisk provides the functions and endpoints of the FaaS environment, and Meteoroid Core abstracts the resources of FIWARE's Subscription function and the function of providing the functions and endpoints of OpenWhisk, centrally manages them, and provides an interface with a RESTful API.
```
Comment (ryo-ma) [inline docs/architecture.md]: I suggest you should link the GitHub page URL to keywords of OpenWhisk.

Comment (ryo-ma) [reply docs/architecture.md]: In addition, you should list this sentence. 

Like the following.

---
OpenWhisk provides the following items.

#### FaaS environment

- Functions
- Endpoints
- 

#### Abstract resources

- FIWARE's Subscription function
- ....
- ....
- 

#### Interface

- [RESTful API](xxxxx)
- [Meteoroid CLI](xxxxx)
- ....

....

File: docs/index.md (hunk 1 above)
Comment (hide-hayashi) [reply docs/index.md]: Thanks. I'll fix the figure name and header.

File: docs/architecture.md (hunk 2 above)
Comment (hide-hayashi) [reply docs/architecture.md]: Thanks I'll fix them

**Review by ryo-ma — APPROVED** [review]

**Comment by ryo-ma** [comment]
LGTM

**Pull request merged**